molecule/default/tasks/log.yml

```diff
@@ -79,6 +79,43 @@
           - "'hello world' in pod_log.log"
           - item == 'hello world' or item == ''
       with_items: '{{ pod_log.log_lines }}'
+
+    - name: Create a job that calculates pi
```

Comment:
You have me ROFL on that test change, but now you need to change the name for this task 😂

Reply:
It's not a very accurate calculation, but it's not too far off

Reply:
"What's pi?"

Oh, around 7 or so...

😆

Reply:
@geerlingguy the task name is now more accurate